### PR TITLE
Linting fix

### PR DESCRIPTION
**Pull request by AronBuzogany**
Solves issue #16

### CODE REVIEW

**Comment by pheyvaer** [comment]
Is the `linting-fix` up to date with the latest things, such as the Solid login?

**Comment by AronBuzogany** [comment]
> Is the `linting-fix` up to date with the latest things, such as the Solid login?

I have merged it locally and there are no ESLint warnings. Only warnings are the webpack warnings which are also mentioned in #2

File: src/workers/worker.js
Comment (pheyvaer) [inline src/workers/worker.js:1]: Wasn't this file already deleted before?

**Comment by pheyvaer** [comment]
> > Is the `linting-fix` up to date with the latest things, such as the Solid login?
> 
> I have merged it locally and there are no ESLint warnings. Only warnings are the webpack warnings which are also mentioned in #2

image

File: src/workers/worker.js
Comment (AronBuzogany) [reply src/workers/worker.js:1]: It indeed was, but this branch checked out a branch that is now merged so all its commits are also present.

**Comment by AronBuzogany** [comment]
> > > Is the `linting-fix` up to date with the latest things, such as the Solid login?
> > 
> > 
> > I have merged it locally and there are no ESLint warnings. Only warnings are the webpack warnings which are also mentioned in #2
> 
> image

As I said I did it locally to avoid unnecessary commits. It is now however pushed.

**Comment by pheyvaer** [comment]
Having it locally merged doesn't do me any good. I mean when I then look at the files changed here on GitHub it shows me changes that have nothing to do with the issue that this PR fixes.

**Review by pheyvaer — APPROVED** [review]

**Pull request merged**